spec/lib/protobuf/generators/field_generator_spec.rb

```diff
@@ -83,19 +92,53 @@
     context 'when the field is an extension' do
       let(:extendee) { 'foo.bar.Baz' }
 
-      specify { expect(subject).to eq "optional :string, :foo_bar, 3, :extension => true\n" }
+      specify { expect(subject).to eq "  optional :string, :foo_bar, 3, :extension => true\n" }
     end
 
     context 'when field is packed' do
       let(:field_options) { { :packed => true } }
 
-      specify { expect(subject).to eq "optional :string, :foo_bar, 3, :packed => true\n" }
+      specify { expect(subject).to eq "  optional :string, :foo_bar, 3, :packed => true\n" }
+    end
+
+    context 'when field is a map' do
```

Comment:
It might also be nice to see a test that goes from the real bytes of a complied proto file to the generated code, like there is in the piecewise-custom-descriptor-options branch. https://github.com/nerdrew/protobuf-gem/commit/9ee9587a189df0557219734e36ce7c11efc48126